config/jobs/kubernetes-sigs/cluster-api/cluster-api-periodics-release-1-3-upgrades.yaml

```diff
@@ -375,50 +375,3 @@ periodics:
     testgrid-tab-name: capi-e2e-release-1-3-1-25-1-26
     testgrid-alert-email: sig-cluster-lifecycle-cluster-api-alerts@kubernetes.io
     testgrid-num-failures-to-alert: "4"
-
-- name: periodic-cluster-api-e2e-workload-upgrade-1-26-latest-release-1-3
```

Comment:
The Kubernetes 1.27 release is ~ 3 weeks after the v1.4 Cluster API release.

Because of that support for Kubernetes 1.27 will likely be backported to v1.4 but not to v1.3

This means we don't have to aim for compatibility between Cluster API v1.3 and Kubernetes 1.27. It's enough to test main (upcoming v1.4) against Kubernetes latest (upcoming 1.27)

Reply:
Same for v1.2 below